src/Common/src/System/Net/Logging/SocketsEventSource.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Net.Sockets;
+
+namespace System.Net
+{
+
+    [EventSource(Name = "Microsoft-System-Net-Sockets",
+        Guid = "e03c0352-f9c9-56ff-0ea7-b94ba8cabc6b",
+        LocalizationResources = "FxResources.System.Net.Sockets.SR")]
```

Comment:
Is this needed to make this work? I'm afraid that it will silently stop working if someone decides to rename the resource file.